app/UiTPAS/UiTPASApiInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UiTPAS;
+
+use App\Keycloak\Realm;
+
+interface UiTPASApiInterface
+{
+    public function addPermissions(Realm $realm, string $organizerId, string $clientId): void;
```

Comment:
I don't think `Realm` is needed in this context. I think the Keycloak Realm concept now leaks into a simple UiTPAS API call.

Reply:
I think the concrete `UiTPASApi` needs to be injected with the `Realm` but not this interface.